docs/docs/installation/pypi.mdx

```diff
@@ -129,21 +121,33 @@ First, start by installing `apache-superset`:
 pip install apache-superset
 ```
 
-Then, you need to initialize the database:
+Superset configurations are stored in a file.  One of these configurations, a `SECRET_KEY`, is required for the application to start. Create your config file:
+```
+touch superset/superset_config.py
```

Comment:
Can Use below command - https://github.com/apache/superset/blob/271510f0a81422c62cceb09245a55d844e749d60/docs/docs/contributing/local-backend.mdx?plain=1#L30

echo "SECRET_KEY='$(openssl rand -base64 42)'" | tee -a superset_config.py

Reply:
Great addition, I've implemented this.